src/linearmixedmodel.jl

```diff
@@ -105,7 +106,7 @@ function LinearMixedModel(
     θ = foldl(vcat, getθ(c) for c in reterms)
     optsum = OptSummary(θ, lbd, :LN_BOBYQA, ftol_rel = T(1.0e-12), ftol_abs = T(1.0e-8))
     fill!(optsum.xtol_abs, 1.0e-10)
-    LinearMixedModel(form, reterms, feterms, sqrt.(convert(Vector{T}, wts)), A, L, optsum)
+    LinearMixedModel(form, reterms, feterms, terms, sqrt.(convert(Vector{T}, wts)), A, L, optsum)
```

Comment:
why does `terms` need to be added here? The field in the struct is `allterms`, which seems to replace the old `terms` field in functionality.

Reply:
The name `terms` was already in place and I just didn't bother to change it to `allterms` here.  `terms` is a local identifier and the name of the field is `allterms`.  I agree that it would make more sense to use `allterms` for both and will make that change.